src/mlpack/tests/linear_regression_test.cpp

```diff
@@ -216,9 +215,9 @@ BOOST_AUTO_TEST_CASE(LinearRegressionTest)
 
   SerializeObjectAll(lr, xmlLr, textLr, binaryLr);
 
-  BOOST_REQUIRE_CLOSE(lr.Lambda(), xmlLr.Lambda(), 1e-8);
-  BOOST_REQUIRE_CLOSE(lr.Lambda(), textLr.Lambda(), 1e-8);
-  BOOST_REQUIRE_CLOSE(lr.Lambda(), binaryLr.Lambda(), 1e-8);
+  REQUIRE(lr.Lambda() == Approx(xmlLr.Lambda()).epsilon(1e-8 / 100));
```

Comment:
I think it may make more sense from readability to have it in 1e-10 format, but I think this is easier to have as the result of automation. Did you use a regex or similar for this PR?

Reply:
Done

Reply:
No I did it manually